apps/pip/stg/base/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+namespace: pip
+patchesStrategicMerge:
+  - ../../identity/pip-azure-identity.yaml
+  - ../../identity/stg.yaml
+  - ../../account-management/pip-account-management.yaml
+  - ../../channel-management/pip-channel-management.yaml
+  - ../../data-management/pip-data-management.yaml
+  - ../../frontend/pip-frontend.yaml
+  - ../../publication-services/pip-publication-services.yaml
+  - ../../subscription-management/pip-subscription-management.yaml
```

Comment:
you don't need to add patches for a base file, the resources already covers that